app/schemas.py

```diff
@@ -284,12 +285,12 @@ class Meta(BaseSchema.Meta):
     def validate_permissions(self, value):
         permissions = [v.permission for v in value]
         for p in permissions:
-            if p not in models.SERVICE_PERMISSION_TYPES:
-                raise ValidationError("Invalid Service Permission: '{}'".format(p))
+            if p not in {e for e in ServicePermissionType}:
```

Comment:
Just checking to make sure that this is matching the was expected before, which I'm assuming was also a set. :-)

Reply:
Python is more efficient with doing the `in` operator with sets than with lists, so that's probably just me automatically upgrading it.